docker/packager/packager

```diff
@@ -223,11 +223,9 @@ def parse_env_variables(
 
     cxx = cc.replace("gcc", "g++").replace("clang", "clang++")
 
-    if package_type == "deb":
+    if package_type != "coverity":
```

Comment:
I am not sure how it is supposed to work. The deb package is not a thing for freebsd and other special binaries. Besides that, coverity is not a thing anymore. I leave it as is at the merge